src/main/java/games/strategy/engine/data/CompositeRouteFinder.java

```diff
@@ -37,7 +37,7 @@ public CompositeRouteFinder(final GameMap map, final HashMap<Match<Territory>, I
     s_logger.finer("Initializing CompositeRouteFinderClass...");
   }
 
-  private static HashSet<Territory> ToHashSet(final Collection<Territory> ters) {
+  private static HashSet<Territory> toHashSet(final Collection<Territory> ters) {
```

Comment:
Consider removing this method entirely.
`new HashSet<>(ters)` achieves the exact same thing